src/delete.ts

```diff
@@ -113,9 +115,12 @@ export class DeleteQuery<
     ]);
   }
 
-  returning<C1 extends keyof (T extends Table<string, infer Columns> ? Columns : never)>(
+  returning<C1 extends keyof (T extends Table<string, infer Columns, unknown> ? Columns : never)>(
     column1: C1,
-  ): DeleteQuery<T, GetReturning<T extends Table<string, infer Columns> ? Columns : never, C1>>;
+  ): DeleteQuery<
+    T,
+    GetReturning<T extends Table<string, infer Columns, unknown> ? Columns : never, C1>
```

Comment:
Unsure if the third type parameter for `Table` in these two lines should be `unknown` or `any`.